src/player.js

```diff
@@ -594,6 +594,30 @@ export default class Player extends FakeEventTarget {
     }
   }
 
+  /**
+   * Get video height.
+   * @returns {?number} - The intrinsic height of the video.
+   * @public
+   */
+  get videoHeight(): ?number {
+    if (this._engine) {
+      return this._engine.videoHeight;
+    }
+    return null;
```

Comment:
I would return 0 so the type can be number

Reply:
I thought undefined was more appropriate actually. But the reason I chose to return null is to be aligned with all other number getters we have in the player. For example - 
https://github.com/kaltura/playkit-js/blob/ee7653f7cae10fcf2e18e581e8661ccf9308173f/src/player.js#L816
https://github.com/kaltura/playkit-js/blob/ee7653f7cae10fcf2e18e581e8661ccf9308173f/src/player.js#L828
https://github.com/kaltura/playkit-js/blob/ee7653f7cae10fcf2e18e581e8661ccf9308173f/src/player.js#L841